templates/etc/logrotate.conf.erb

```diff
@@ -5,6 +5,12 @@
 # rotate log files weekly
 weekly
 
+<% if @lsbmajdistrelease and (scope.function_versioncmp([@lsbmajdistrelease, '14']) >= 0) -%>
```

Comment:
You want to check both to insure that this change doesn't happen on distros other than Ubuntu.  I.e. this would incorrectly be applied to Fedora releases > 14.